src/MySQL.jl

```diff
@@ -249,6 +284,7 @@ Connect to a MySQL database with provided `host`, `user`, and `passwd` positiona
   * `option_file::String`: the argument is interpreted as a path to a custom option file, and only that option file is read.
   * `read_default_group::Bool`: only the default option groups are read from specified option file(s)
   * `option_group::String`: it is interpreted as a custom option group, and that custom option group is read in addition to the default option groups.
+  * `enable_cleartext_plugin::Bool`: This option is supported to be compatible with MySQL client libraries. MySQL client libraries use this option to determine whether the mysql_clear_password authentication plugin can be used.
```

Comment:
Here's the mariadb docs as to why I didn't originally include this option:
```
For MySQL compatibility, MariaDB Connector/C also allows applications to set the MYSQL_ENABLE_CLEARTEXT_PLUGIN option with the mysql_optionsv function. However, this option does not actually do anything in MariaDB Connector/C, because the mysql_clear_password client authentication plugin is always enabled for MariaDB clients and client libraries.
```
So yeah, I don't really mind including it here, but it sounds like it doesn't really do anything w/ mariadb client (which MySQL.jl uses under the hood)

Reply:
@quinnj Got it, thank you! I'd thought adding it was what made my use case work, but it must have been something else -- I just tried without the option and it did work. Removing the extra code here.